tests/dehumanize_my_test.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+from chai import Chai
+
+import arrow
+from arrow import locales
+
+utc = arrow.utcnow()
```

Comment:
You can make use of `nosetest` setup functions to prevent the use of global variables. Check out other test files to see how this is done :).